### DISCUSSION TITLE
使用示例anndata文件启动 Annotate 报错

### DISCUSSION THREAD

**Discussion by woaichitudou123** [chanzuckerberg/cellxgene#2643] [Q&A]
以下是我的报错信息
<img width="1211" alt="截屏2023-11-26 23 48 13" src="https://github.com/chanzuckerberg/cellxgene/assets/152083261/81e0343c-5d4c-4e47-adef-63a83eb341bd">
新手帮忙看一哈如何解决

**Comment by MaximilianLombardo** [comment]
Answered [here](https://github.com/chanzuckerberg/cellxgene/issues/2642), closing this discussion to avoid duplication.
